### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by RafalZakrzewski**
Clean code commit

### CODE REVIEW

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    public ProductData getprddata() {
+        return products; //return here a state of shipping
+    }
+
+    //getter for clients
+    public void addProduct(String name) {
+        client.setName(name);
+    }
+
+//    private Boolean isFreeShiping() {
```
Comment (zakrzewski-rafal) [inline src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: Remove commented code

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    public Client getBuyer() {
+        return client;
+    }
+
+    public ProductData getprddata() {
+        return products; //return here a state of shipping
+    }
+
+    //getter for clients
```
Comment (zakrzewski-rafal) [inline src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: Comment not needed

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package java.com.infoshareacademy.cleancode.rafalzakrzewski;
+
+import java.com.infoshareacademy.codereview.Client;
+import java.math.BigDecimal;
+
+/** Shopping cart create */
```
Comment (zakrzewski-rafal) [inline src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: Comment not needed

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+public class ShoppingCartCreate {
+    private ProductData products;
+    private Client client;
+
+    /** Constructor */
+    public ShoppingCartCreate(Client client) {
+        this.client = client;
+    }
+
+    public Client getBuyer() {
```
Comment (zakrzewski-rafal) [inline src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: Change to getClient()

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        return client;
+    }
+
+    public ProductData getprddata() {
```
Comment (zakrzewski-rafal) [inline src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: Change to getProductData

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    public void addProduct(String name) {
+        client.setName(name);
+    }
+
+//    private Boolean isFreeShiping() {
+//        return true;
+//    }
+
+    public BigDecimal getTotalPrice(boolean showVat) {
+        BigDecimal totalPrice = BigDecimal.valueOf(Double.valueOf(new java.text.DecimalFormat("#.##").format(products.getPrice().add(products.getVat()))));
```
Comment (zakrzewski-rafal) [inline src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: This line is too loong

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ProductData.java [hunk 7]
```diff
@@ -0,0 +1,39 @@
+package java.com.infoshareacademy.cleancode.rafalzakrzewski;
+
+import java.math.BigDecimal;
+
+public class ProductData {
```
Comment (zakrzewski-rafal) [inline src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ProductData.java]: Please improve formatting ot this class

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java (hunk 1 above)
Comment (RafalZakrzewski) [reply src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: I agree

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java (hunk 2 above)
Comment (zakrzewski-rafal) [reply src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: Removed

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java (hunk 4 above)
Comment (zakrzewski-rafal) [reply src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: True

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java (hunk 5 above)
Comment (zakrzewski-rafal) [reply src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: Done

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ProductData.java (hunk 7 above)
Comment (zakrzewski-rafal) [reply src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ProductData.java]: Yes

File: src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java (hunk 5 above)
Comment (zakrzewski-rafal) [reply src/java/com/infoshareacademy/cleancode/rafalzakrzewski/ShoppingCartCreate.java]: ok

**Pull request merged**